### PR TITLE
[Fix #233] Add more trailing comma rules

### LINKED ISSUE #233: Configure other trailing comma cops
Fix other cops with trailing comma as for this issue https://github.com/datarockets/datarockets-style/issues/218

### PR DESCRIPTION
**Pull request by paydaylight**
### Added more trailing comma cops as in [#218](https://github.com/datarockets/datarockets-style/issues/218):
* Added `EnforcedStyleForMultiline: comma`  for `Style/TrailingCommaInArguments` and `Style/TrailingCommaInArrayLiteral`.
* Did not enable `Style/TrailingCommaInBlockArgs` because of safety warnings and unsure usage purpose

Before you submit a pull request, please make sure you have to follow:

- [x] read and know items from the [Contributing Guide](https://github.com/datarockets/datarockets-style/blob/master/CONTRIBUTING.md#pull-requests)
- [x] add a description of the problem you're trying to solve (short summary from related issue)
- [x] verified that cops are ordered by alphabet
- [x] add a note to the style guide docs (if it needs)
- [x] add a note to the changelog file
- [x] the commit message contains the number of the related issue (if it presents)
  and word `Fix` if this PR closes related issue
- [x] squash all commits before submitting to review

### CODE REVIEW

**Review by texpert — APPROVED** [review]

File: CHANGELOG.md [hunk 1]
```diff
@@ -7,6 +7,7 @@ The format is described in [Contributing notes](CONTRIBUTING.md#changelog-entry-
 ## Changed
 
 * Change ```Datarockets::Style``` module to ```DatarocketsStyle``` ([@paydaylight](https://github.com/paydaylight))
+* [#233](https://github.com/datarockets/datarockets-style/issues/233) Setup `EnforcedStyleForMultiline` for `Style/TrailingCommaInArguments` and `Style/TrailingCommaInArrayLiteral` rules. ([@paydaylight](https://github.com/paydaylight))
```
Comment (roman-dubrovsky) [inline CHANGELOG.md:10]:
```suggestion
* [#233](https://github.com/datarockets/datarockets-style/issues/233) Setup `EnforcedStyleForMultiline` for `Style/TrailingCommaInArguments` and `Style/TrailingCommaInArrayLiteral` rules. ([@paydaylight][])
```

**Review by roman-dubrovsky — COMMENTED** [review]
Can you rebase this PR to the latest master?

**Comment by roman-dubrovsky** [comment]
> Did not enable Style/TrailingCommaInBlockArgs because of safety warnings and unsure usage purpose

agreed about `unsure usage purpose` but @paydaylight what did you mean when wrote `because of safety warnings` ?

**Comment by paydaylight** [comment]
@roman-dubrovsky cop is labeled `unsafe`, that's what I meant

File: CHANGELOG.md [hunk 2]
```diff
@@ -4,19 +4,21 @@ The format is described in [Contributing notes](CONTRIBUTING.md#changelog-entry-
 
 ## master
 
+* [#233](https://github.com/datarockets/datarockets-style/issues/233) Setup `EnforcedStyleForMultiline` for `Style/TrailingCommaInArguments` and `Style/TrailingCommaInArrayLiteral` rules. ([@paydaylight][])
```
Comment (roman-dubrovsky) [inline CHANGELOG.md:7]:
```suggestion
### Changed

* [#233](https://github.com/datarockets/datarockets-style/issues/233) Setup `EnforcedStyleForMultiline` for `Style/TrailingCommaInArguments` and `Style/TrailingCommaInArrayLiteral` rules. ([@paydaylight][])
```

File: lib/datarockets_style/version.rb [hunk 3]
```diff
@@ -1,3 +1,3 @@
 module DatarocketsStyle
-  VERSION = "1.1.0".freeze
+  VERSION = "1.0.0".freeze
```
Comment (roman-dubrovsky) [inline lib/datarockets_style/version.rb:2]: same about version

**Review by roman-dubrovsky — CHANGES_REQUESTED** [review]

**Comment by paydaylight** [comment]
@roman-dubrovsky fixed

**Review by roman-dubrovsky — APPROVED** [review]

**Pull request merged**